xapian-core/cluster/euclidian_sim.cc

```diff
@@ -40,86 +40,10 @@ EuclidianDistance::get_description() {
     return "Euclidian Distance metric";
 }
 
+/** TODO
+ *  Implement the euclidian distance metric for the current API
+ */
```

Comment:
Rather than do this, maybe just remove the euclidian distance metric completely? This also feels like this commit (ec28b37) is actually a fixup to the previous one, and should be noted as such to ensure it gets collapsed before merging.